### DISCUSSION TITLE
求十字军之王2的appimage版本（如钢铁雄心4的）

### DISCUSSION THREAD

**Discussion by zqtg-sudo** [Gamuxorg/bbs#271] [投递和请求游戏]
游戏名称：十字军之王2
游戏名称(英文)：Crusader Kings 2
下载地址 :（如果有的话）：站里有他的.sh文件版本，之所以要是因为我搞了一整天还是搞不懂如何运行他，只能求帮他打包一下了。。。

**Comment by shouhuanxiaoji** [comment]
好的，尽快打包

**Comment by hyh19962008** [comment]
这个游戏打包之后无法运行，所以没有打包
